libs/experimental/langchain_experimental/sql/base.py

```diff
@@ -173,7 +173,7 @@ def _call(
                 sql_cmd = checked_sql_command
 
             _run_manager.on_text("\nSQLResult: ", verbose=self.verbose)
-            _run_manager.on_text(result, color="yellow", verbose=self.verbose)
+            _run_manager.on_text(str(result), color="yellow", verbose=self.verbose)
```

Comment:
> Can we just cast the output of SQLDatabase wherever it's passed to `on_text` instead of changing the `on_text` method.

@baskaryan: Your suggestion has been implemented here.

Reply:
This seems fine to me